src/go/rdctl/pkg/factoryreset/factory_reset_windows.go

```diff
@@ -1,59 +1,41 @@
-/*
-Copyright © 2021 SUSE LLC
+//go:build windows
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// Package reset implements factory reset for Windows.
-package reset
+package factoryreset
 
 import (
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path"
 	"strings"
+	"syscall"
 	"unsafe"
 
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sys/windows"
+	"golang.org/x/text/encoding/unicode"
 )
 
-const (
-	appName = "rancher-desktop"
-)
-
-// Factory reset deletes any Rancher Desktop user data.
-func FactoryReset(keepSystemImages bool) error {
-	dirs, err := getDirectoriesToDelete(keepSystemImages)
+func DeleteWindowsData(keepSystemImages bool, appName string) error {
+	dirs, err := getDirectoriesToDelete(keepSystemImages, appName)
 	if err != nil {
 		return err
 	}
 	for _, dir := range dirs {
 		logrus.WithField("path", dir).Trace("Removing directory")
-		if err := os.RemoveAll(dir); err != nil {
-			if !errors.Is(err, os.ErrNotExist) {
-				return fmt.Errorf("could not remove %s: %w", dir, err)
-			}
+		err := os.RemoveAll(dir)
+		if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
`os.RemoveAll` is documented to return `nil` if the path does not exist; we shouldn't check against `os.ErrNotExist` here. (If we see it, something else went wrong, like a disappearing NFS mount.)

Reply:
ok